<?xml version="1.0"?>
<launch>
    <arg name="real_world" default="false"/>
    <node type="detector.py" name="detector" pkg="martian-mines" output="screen">
        <param name="real_world" value="$(arg real_world)"/>
        <param name="nn_model_path" value="$(find martian-mines)/nn_models/n_20ep.pt"/>
        <remap if="$(arg real_world)" from="/uav0/camera/image_raw" to="/uav0/color/image_raw"/>
    </node>


    <!-- launch realsense camera node -->
    <group if="$(arg real_world)">
    <include file="$(find martian-mines)/launch/realsense.launch"></include>

    <!-- launch mavros -->
    <include file="$(find mavros)/launch/px4.launch">
        <arg name="fcu_url" value="udp://:14550@localhost:14550" />
    </include>

    </group>
</launch>